TrkHitReco/fcl/prolog.fcl

```diff
@@ -147,6 +147,14 @@ TrkHitReco : { @table::TrkHitReco
       DebugLevel           : 0
    }
 }
+
+TrkHitReco : { @table::TrkHitReco
```

Comment:
My apologies I must have mis-spoke, this stanza belongs in TrkReco, not TrkHitReco.